Geometry/Records/interface/CaloTowerGeometryRecord.h

```diff
@@ -27,6 +28,7 @@ class CaloTowerGeometryRecord :
                 IdealGeometryRecord,
 		CaloTowerAlignmentRcd, 
 		CaloTowerAlignmentErrorRcd,
```

Comment:
Can this dependency on *AlignementErrorRcd be moved to the *AlignementErrorExtendedRcd?